Intersect.Client/Core/Graphics.cs

```diff
@@ -364,6 +364,11 @@ public static void DrawInGame()
                         entity.DrawName(null);
                         if (entity.GetType() != typeof(Event))
                         {
+                            // NpcTags
+                            if (Options.Npc.ShowNpcTags && entity.GetType() != typeof(Player))
```

Comment:
As I noted below these checks should be moved into the Draw method call.

The draw method should also be renamed to be generic since it's virtual.